### PR TITLE
feat: add errorcodes to validationexception

### PR DESCRIPTION
**Pull request by pgallik**

### CODE REVIEW

File: src/StreetNameRegistry.Api.BackOffice/StreetName/StreetNameController.cs [hunk 1]
```diff
@@ -105,29 +105,42 @@ public class StreetNameController : ApiBusController
...
+                        "'Straatnamen' is missing an official or facility language."),
 
                     _ => new ValidationException(new List<ValidationFailure> { new ValidationFailure(string.Empty, exception.Message) })
                 };
             }
         }
+
+        private ValidationException CreateValidationException(string errorCode, string propertyName, string message)
+        {
+            var failure = new ValidationFailure(propertyName, message)
```
Comment (ArneD) [inline src/StreetNameRegistry.Api.BackOffice/StreetName/StreetNameController.cs:135]: Nice refactoring!

**Comment by ArneD** [comment]
Can you update `api` package with the Invalid Json code, then update the package in this PR?

File: src/StreetNameRegistry.Api.BackOffice/StreetName/StreetNameController.cs (hunk 1 above)
Comment (pgallik) [reply src/StreetNameRegistry.Api.BackOffice/StreetName/StreetNameController.cs:135]: Thanks!

**Comment by pgallik** [comment]
> Can you update `api` package with the Invalid Json code, then update the package in this PR?

I will leave it open and amend the commit :)

**Comment by pgallik** [comment]
@ArneD api has been updated

**Pull request merged**